spec/features/draws/draw_suite_update_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Draw suite update' do
+  let(:draw) { FactoryGirl.create(:draw_with_members, suites_count: 1) }
+  let!(:removed_suite) { draw.suites.first }
+  let(:other_draw) { FactoryGirl.create(:draw_with_members, suites_count: 1) }
+  let!(:other_draw_suite) { other_draw.suites.first }
```

Comment:
This is because without the bang it only executes when called, but the bang executes prior to the tests. Since `other_draw_suite` refers to `other_draw`, by applying the bang there it effectively causes both to execute.

This was necessary since the views require the data to be persisted prior to rendering and we only refer to the variables in the assertion.